src/main/java/org/apache/maven/plugins/gpg/SignAndDeployFileMojo.java

```diff
@@ -257,6 +219,15 @@ public class SignAndDeployFileMojo extends AbstractGpgMojo {
     @Parameter(property = "files")
     private String files;
 
+    /**
+     * @since 3.0.0
+     */
+    @Parameter(defaultValue = "${session}", readonly = true, required = true)
```

Comment:
why not @Component?

Reply:
Coz did not want to mix in that change, so I did "as it was for the rest". anyway, pushed now a change that moves ALL to component.